browser/src/Plugins/Api/Ui.ts

```diff
@@ -0,0 +1,14 @@
+import { Icon, IconProps, IconSize } from "../../UI/Icon"
+
+export class Ui {
+    constructor(private _react: any) {
+    }
+
+    public createIcon(props: IconProps): any {
+        return this._react.createElement(Icon, props)
```

Comment:
Cool, thanks for including both this and `Oni.services.git` in our API surface. Great idea!